vendor/geometry-utils/src/worker-flow/pair-flow.ts

```
@@ -0,0 +1,1100 @@
+    const x = (b1 * c2 - b2 * c1) / denom;
+    const y = (a2 * c1 - a1 * c2) / denom;
+
+    // lines are colinear
+    /* var crossABE = (E.y - A.y) * (B.x - A.x) - (E.x - A.x) * (B.y - A.y);
+		var crossABF = (F.y - A.y) * (B.x - A.x) - (F.x - A.x) * (B.y - A.y);
+		if(_almostEqual(crossABE,0) && _almostEqual(crossABF,0)){
+			return null;
+		}*/
+
+    return !(
+        !(isFinite(x) && isFinite(y)) ||
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)